### DISCUSSION TITLE
How to get character FBX file

### DISCUSSION THREAD

**Discussion by HotObsidian** [naverz/zepeto-studio-global#71] [Q&A]
I want to make a set of animations in modeling software based on zepeto's character, can you provide the model and skeleton of the character.

**Comment by naverz-mkao** [comment]
Hi HotObsidian!

You can get the Zepeto base model files on this page:

https://studio.zepeto.me/guides/creating-your-item-3d-modelers-1

Here's the link to the Zepeto Creator Base Set file:

[creatorBaseSet_zepeto.zip](https://media-contentful.zepeto.io/ttn7jks47ozr/cf9E63wK88bsFcsdb4EueIM/G0ml73wK87j5a4e0b9cf032953487503607f54ddac5sFcsNlcqVydH/creatorBaseSet_zepeto.zip)

**Reply by HotObsidian** [reply]
thx~

**Reply by HotObsidian** [reply]
Hello, our 3D designer used the file creatorBaseSet_zepeto.fbx to make several action files in maya, but when the character uses these actions, the character will only do crouching action. Do you know why?

**Comment by naverz-mkao** [comment]
Hi HotObsidian!

Setting animation type to "Humanoid" should fix this issue.

Take a look at the explanation in this discussion: https://github.com/naverz/zepeto-studio-global/discussions/44

Let me know if you still are having issues.

**Reply by HotObsidian** [reply]
It worked, thanks a lot
